README.md

```diff
@@ -47,7 +47,7 @@ We present Crackling, a new method for whole-genome identification of suitable C
 1. Extract off-target sites:
 
     ```
-    python3.7 ExtractOfftargets.py <output-file>  (input-files... | input-dir>)
+    python3.7 extractOfftargets.py <output-file>  (input-files... | input-dir>)
```

Comment:
I've noticed a small typo in this line - the disconnected `>`.

I suggest reformatting this to:

```suggestion
    python3.7 extractOfftargets.py <output-file> {<input-file> ... | <input-dir>}
```

This would need to be updated in `extractOfftargets.py` too

However, I'm not 100% sure on convention. I looked at:
- [Utility argument syntax](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html)
- [Wikipedia - command description syntax](https://en.wikipedia.org/wiki/Command-line_interface#Command_description_syntax)
- For Windows, [Command-Line syntax key](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2012-R2-and-2012/cc771080(v=ws.11)?redirectedfrom=MSDN)


Any thoughts on this?

Reply:
I think your proposed reformatting is fine. I think it is the most likely to be understood by people across different systems.

Reply:
Hi @jakeb1996,
I have updated the README appropriately.
Resolving this request.